packages/blade/src/tokens/theme/theme.d.ts

```diff
@@ -102,9 +125,9 @@ export type Colors = {
     background: Record<'level1' | 'level2' | 'level3', ColorContrast>;
     border: Record<'normal' | 'subtle', ColorContrast>;
     text: Record<TextTypes, ColorContrast>;
-    action: {
-      icon: Pick<ActionVariants, 'link'>;
-    };
+    // action: {
+    //   icon: Pick<ActionStates, 'link'>;
+    // };
```

Comment:
Removing `surface.action.icon` types since we won't have any standalone clickable icon on any of the surfaces. It would always be a button icon for which the relevant types exist

Reply:
confirm with Saurav once and then delete this entirely